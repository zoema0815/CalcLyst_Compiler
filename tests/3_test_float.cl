int main () {
    float x;
    float y;
    x = 4.1;
    y = 2.1;
    printFloat(x);
    printFloat(y);
    printFloat(x + y);
    printFloat(x - y);
    printFloat(x * y);
    printFloat(x / y);
    return 0;
}