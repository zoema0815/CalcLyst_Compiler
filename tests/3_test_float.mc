int main () {
    float x;
    x = 3.1;
    printFloat(x);
    return 0;
}