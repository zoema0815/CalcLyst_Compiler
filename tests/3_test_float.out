4.1
2.1
6.2
2
8.61
1.95238
